boltdesignsystem/bolt
# DS-603 and DS-611 docs layout enhancements

## Jira

https://pegadigitalit.atlassian.net/browse/DS-603
https://pegadigitalit.atlassian.net/browse/DS-611

## Summary

Updated top level link styles and added icon support for links in Side Nav component.

## Details

1. Top level links are semibold
2. Hyphenation is removed
3. Icon support added for links to support external link

## How to test

Run the branch locally and check the Side Nav docs and the Community Docs Layout demo page. Make sure all changes are displaying as written above.

## Release notes

### Visual changes
#### Side Nav
- Top level links are now semibold.
- Side Nav links and text will no longer hyphenate.
